### ISSUE TITLE
MoonScript transformer

### ISSUE THREAD

**Issue by safazi** [siffiejoe/lua-amalg#14]
Low hanging fruit.
Require the user to have `moonc` in their path - it isn't your responsibility to handle that.
Just take the result of `os.execute`ing `moonc`.
I'll make the PR later.

**Comment by safazi** [comment]
Reading more, I realized this would only work for 5.1 because amalg uses `package.searchpath`.
Maybe not so low hanging anymore.

**Comment by siffiejoe** [comment]
We did some parallel work here, apparently.

**Comment by siffiejoe** [comment]
Extending `package.path` in the transformation plugin was a good idea!

**Comment by safazi** [comment]
> 
> 
> Extending `package.path` in the transformation plugin was a good idea!

Yeah, it was the quickest solution I could think of for the searchpath problem.
Not related to this issue, but is there anything that can be done about globbing?
I'd love to be able to specify an entire directory to include.
Example: `amalg.lua -s main.lua src/*.lua`
